.tool-versions

```diff
@@ -1,3 +1,3 @@
-hugo extended_0.101.0
+hugo extended_0.122.0
```

Comment:
So, I have been trying to decide if it is a problem that this value does not match the [min Hugo Version](https://github.com/medic/cht-docs/blob/b8e560d9f9b1f58efbe3e61b6bb512af562b3a97/config.toml#L74) in the config.toml.  Ultimatly, I think this could create problems in the future since the Hugo version [used for CI testing](https://github.com/medic/cht-docs/blob/b8e560d9f9b1f58efbe3e61b6bb512af562b3a97/.github/workflows/ci.yml#L26) is based on this `.tool-versions` value.

Currently the min version is set to `0.93.0`.  But if I am running `0.101.0` locally and use a feature that requires `0.101.0`, then the CI will not catch it and anyone actually using `0.93.0` will be broken....

Perhaps we should just remove the min version from the `config.toml` and just say that the value here is the source of truth for the supported version of Hugo...

Reply:
yo!! I knew I picked you for good reason to review my PR.  totally didn't dive this deep - thank you!

fixed in 64fa0eed